protocol/examples/messageData/federation_result/federation_result.json

```diff
@@ -0,0 +1,12 @@
+{
+    "object": "federation",
+    "action": "result",
+    "status": "success",
+    "challenge": {
+        "data": "eyJvYmplY3QiOiJmZWRlcmF0aW9uIiwiYWN0aW9uIjoiY2hhbGxlbmdlIiwidmFsdWUiOiJlYmEzZTI0ZWZjZDBiNTNmYTY5OTA4YmFkNWQxY2I2OTlkNzk4MGQ5MzEwOWRhMGIyYmZkNTAzN2MyYzg5ZWUwIiwidGltZXN0YW1wIjoxNzEzMzg1NTY4fQ==",
+        "sender": "zXgzQaa_NpUe-v0Zk_4q8k184ohQ5nTQhBDKgncHzq4=",
+        "signature": "BILYwYkT5tOBL4rCD7yvhBkhAYqRXOI3ajQ2uJ1gAk-g6nRc38vMMnlHShuNCQ3dQFXYZPn37cCFelhWGjY8Bg==",
+        "message_id": "sD_PdryBuOr14_65h8L-e1lzdQpDWxUAngtu1uwqgEI=",
+        "witness_signatures": []
+    }
+}
```

Comment:
This example doesn't seems to match a full success result message. There is no public_key field.
Maybe we should create two correct example, one for success and one for failure. And add this to the filename.

Reply:
i added one correct positive case now, but you're right there should probably also be one for failure